Makefile

```diff
@@ -32,6 +32,13 @@ install:
 	${GO_BUILD_ENVVARS} go install \
 		-ldflags "-X main.version=${VERSION} -X main.commitHash=${COMMIT_HASH}"
 
+format:
+# Add new paths        find . \( -path './vendor' -o -path <path> \) -prune -o -type f -iname '*.go' -print
+	@for gofile in $$(find . -path './vendor' -prune -o -type f -iname '*.go' -print); do \
+			echo "formatting" $$gofile; \
```

Comment:
We are going to end up with a lot of files - we shouldn't echo every file here or the build output is going to be large. I would take this echo out.

It would be nice if we could echo the names of files that actually changed due to the formatting. That would alert the dev that one of his files changed due to running the build.  But this isn't required. If its easy to do, do it. (I think it is easy - I think the exit code of gofmt might tell us if it changed the file or not???)